### DISCUSSION TITLE
Sticky playground in the component Storybook

### DISCUSSION THREAD

**Discussion by nerdrun** [SchwarzIT/onyx#1031] [Ideas]
It would be great to get the interactive playground `sticky` on the component story so that a user doesn't need to scroll up and down to see the result after adjusting the property values.

For instance the link component story shows the playground where a user can see the result interactively like below
<img width="1054" alt="Screenshot 2024-04-26 at 12 39 27 PM" src="https://github.com/SchwarzIT/onyx/assets/23328939/735974bf-0f12-4eee-ad72-d5ba2db2f435">

However, a user should keep scrolling up and down to see the result after adjusting the property value.
<img width="1044" alt="Screenshot 2024-04-26 at 12 39 32 PM" src="https://github.com/SchwarzIT/onyx/assets/23328939/8cd0350d-c822-49a9-8be9-c6f4c6f89711">

It would be very comfortable that the playground part is sticky so that the user doesn't need to scroll up and down to see the result :)

**Comment by larsrickert** [comment]
@nerdrun If you select a specific story in the sidebar on the left, there are sticky controls at the bottom :)
![image](https://github.com/SchwarzIT/onyx/assets/67898185/3d864e83-81ce-4564-8728-6ad651ace711)

For the first "Docs" page, I don't think Storybook supports this currently.

We are also currently working on changes regarding the documentation

**Reply by nerdrun** [reply]
Thanks for the information. Unless the first "Docs" page, each page has the feature ;)
